src/@types/vscode.proposed.treeItemCheckbox.d.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+
+	export class TreeItem2 extends TreeItem {
+		/**
+		 * [TreeItemCheckboxState](#TreeItemCheckboxState) of the tree item.
+		 */
+		checkboxState?: TreeItemCheckboxState | { state: TreeItemCheckboxState; tooltip?: string };
+	}
```

Comment:
You probably don't want to push this vscode proposed API into the repo, right?

Reply:
All the other API proposals are are in this repo (https://github.com/microsoft/vscode-pull-request-github/tree/main/src/%40types). I can them all to the .gitignore, but for now I'm just going to push it.